app/models/assignment.rb

```diff
@@ -914,6 +915,28 @@ def has_peer_review_assignment?
     not pr_assignment.nil?
   end
 
+  def create_peer_review_assignment_if_not_exist
+    if has_peer_review and Assignment.where(parent_assignment_id: id).empty?
+      peerreview_assignment = Assignment.new
+      peerreview_assignment.parent_assignment = self
+      peerreview_assignment.submission_rule = NoLateSubmissionRule.new
+      peerreview_assignment.assignment_stat = AssignmentStat.new
+      peerreview_assignment.token_period = 1
+      peerreview_assignment.unlimited_tokens = false
+      peerreview_assignment.short_identifier = short_identifier + 'PR'
```

Comment:
No great ideas. Make the suffix `_pr`. Validation will fail on conflict, which is okay for now.